platform.md

```diff
@@ -267,7 +267,7 @@ To rebase an app image a platform MUST execute the `/cnb/lifecycle/rebaser` or p
  
 #### Launch
 `/cnb/lifecycle/launcher` is responsible for launching user and buildpack provided processes in the correct execution environment.
-`/cnb/lifecycle/launcher` SHALL be the `ENTRYPOINT` for all app images.
+`/cnb/lifecycle/launcher`, or a symlink to it (see [exporter outputs](#outputs-4)), SHALL be the `ENTRYPOINT` for all app images.
```

Comment:
Personally I wonder if this level of detail is necessary. Couldn't this statement be true for almost any other binary/file at any path declared in this document?

Reply:
@jromero I believe this was added in response to a specific request by someone who found the current entrypoint (a symlink) surprising, specifically because this wasn't called out in the spec https://github.com/buildpacks/spec/pull/188

Reply:
Definitely a nit but wouldn't a general statement in the spec saying something like "any file may be a symbolic link except where noted" be more accurate? The current statement makes it seem like others can't/shouldn't ever be symbolic links.

Reply:
While a symlink is always valid imo what makes this case special and worth explicitly calling out is that we explicitly create symlinks with meaningful names that affect the behavior of the launcher.